colors/material.vim

```diff
@@ -189,6 +189,21 @@ call s:HL('jsTemplateExpression', s:red, '', '')
 call s:HL('jsThis', s:red, '', '')
 call s:HL('jsUndefined', s:orange, '', '')
 
+" Golang 
+call s:HL('goComment', s:gray5, '', '')
```

Comment:
Is there a reason [this line](https://github.com/kaicataldo/material.vim/pull/11/files#diff-693a3fb1845dd2c150138d65ec2fe8d6R101) doesn't cover comments in Golang? What Golang language plugin are you using?

Also, would you mind updating it to `s:gray4` so it matches?

Reply:
I'm currently using `vim-go` but that highlighting is probably redundant. I'll remove it and see if it still works properly, if not I'll update it to `s:gray4`